### ISSUE TITLE
Acc and training time

### ISSUE THREAD

**Issue by Leo021017** [Jeffkang-94/pytorch-adversarial-attack#4]
Hello! I ran your code to train a model by PGD on WideResNet, which you provide, but the accuracy is far less than your provided pre-trained model, and the training is too slow and time-consuming, could you please tell me the reason? Thank you a lot!

**Comment by SueeH** [comment]
Same problem, train wideResNet by FGSM(or PGD) with cifar10,  only 10% acc, while train clean data, acc same with pre-trained model.
Any solution?

**Comment by SueeH** [comment]
> Same problem, train wideResNet by FGSM(or PGD) with cifar10, only 10% acc, while train clean data, acc same with pre-trained model. Any solution?

put 
self.optimizer.zero_grad()
after
x_adv = self.attacker(x,y)
acc increased
